.github/workflows/pr-realm-js.yml

```diff
@@ -580,27 +570,98 @@ jobs:
         id: mocha-env
         run: echo "context=syncLogLevel=warn,longTimeoutMs=${{ env.LONG_TIMEOUT }},baseUrl=${{ steps.baas.outputs.baas-url }}" >> $GITHUB_OUTPUT
 
-      - name: Run ${{matrix.variant.target}} (${{ matrix.variant.os}} / ${{ matrix.variant.environment }})
-        if: ${{ matrix.variant.os != 'android' && matrix.variant.os != 'ios' }}
-        env:
-          MOCHA_REMOTE_CONTEXT: ${{ steps.mocha-env.outputs.context }}
-        # The non react native environments should not take so long
-        timeout-minutes: 60
-        run: ${{ env.wrapper }} npm run ${{ matrix.variant.target}} --prefix integration-tests/environments/${{ matrix.variant.environment }}
-
-      - name: Run ${{matrix.variant.target}} (${{ matrix.variant.os}} / ${{ matrix.variant.environment }})
-        if: ${{ matrix.variant.os == 'ios' }}
+      # Because the Xcode project injects its own CCACHE_CONFIGPATH,
+      # the configuration set by the ccache action doesn't propagate.
+      - name: Expose Ccache config to Xcode
+        run: |
+          echo "CCACHE_DIR=`ccache --get-config cache_dir`" >> $GITHUB_ENV
+          echo "CCACHE_MAXSIZE=`ccache --get-config max_size`" >> $GITHUB_ENV
+          echo "CCACHE_COMPILERCHECK=`ccache --get-config compiler_check`" >> $GITHUB_ENV
```

Comment:
This was the fix for the main issue.
- The React Native ccache scripts use `CCACHE_CONFIGPATH`: https://github.com/facebook/react-native/blob/f7fe68861a52a0428698a9352d39a6e39afefe42/packages/react-native/scripts/xcode/ccache-clang.sh#L12
- This makes ccache disregard the system-wide configuration, specifically the `cache_dir` which is unexpected to the `hendrikmuhs/ccache-action@v1` action.